vignettes/admiral.Rmd

```diff
@@ -163,120 +97,157 @@ and `TRTSTMF` variables.
 Functions which derive a dedicated parameter start with `derive_param_` followed by
 the parameter name, e.g., `derive_param_bmi()` derives the `BMI` parameter.
 
-# Input and Output
-
-It is expected that the input dataset is not grouped. Otherwise an error is
-issued.
-
-The output dataset is ungrouped. The observations are not ordered in a dedicated
-way. In particular, the order of the observations of the input dataset may not
-be preserved.
-
-# Computations
+## Example: Adding Variables
 
-[Computations](../reference/index.html#computation-functions-for-vectors) expect vectors as
-input and return a vector. Usually these computation functions can not be used with `%>%`. These functions can be used in expressions like
-`convert_dtc_to_dt()` in the derivation of `FINLABDT` in the
-example below:
+Below we can see an example call to one of the most common derivation functions,
+`derive_vars_merged()`. This function adds variable(s) to the input dataset based on the 
+contents of another dataset. In this example, we add the treatment start datetime and 
+corresponding imputation flag (`EXSTTMF`) to `adsl` by identifying the first record
```

Comment:
```suggestion
corresponding imputation flag (`EXSTTMF`) to `ADSL` by identifying the first record
```
I think we do upper case for variables and datasets

Reply:
here i was referencing the actual `adsl` object in my code rather than the general `ADSL` dataset

Reply:
oh i see.  okay - do we do this in other ADAM vignettes??  I guess it is no biggie